clap_complete/tests/testsuite/engine.rs

```diff
@@ -682,7 +682,7 @@ fn suggest_multi_positional() {
     let mut cmd = Command::new("dynamic")
         .arg(
             clap::Arg::new("positional")
-                .value_parser(["pos_1, pos_2, pos_3"])
```

Comment:
Please either revert this commit or switch to what I suggested in https://github.com/clap-rs/clap/pull/5949#discussion_r1999363225

Reply:
In the scenario I gave, we would still have multiple options for `positional-1`

Reply:
Eh, minor enough at this point